### ISSUE TITLE
flags are ignored on project meta creation

### ISSUE THREAD

**Issue by adrianschroeter** [openSUSE/open-build-service#7844]
As @lethliel found out we loose flags on project creation. I am relative sure that this is a regression and it should be fixed before 2.10.

Please note that this is only the case when creating a project, not when updating an existing one.

How to Reproduce
===============
```
1. osc api -X DELETE /source/home:adrianSuSE:test   # just to be sure
2. # cat w
<project name="home:adrianSuSE:test">
  <title>AdrianSuSE's Home Project</title>
  <description>My playground, just for me</description>
  <person userid="adrianSuSE" role="bugowner"/>
  <person userid="adrianSuSE" role="maintainer"/>
<build><disable/></build>

  <repository name="openSUSE_Leap_15.1">
    <path project="openSUSE:Leap:15.1" repository="standard"/>
    <arch>x86_64</arch>
  </repository>
</project>
3. osc api /source/home:adrianSuSE:test/_meta -T w
4. osc api /source/home:adrianSuSE:test/_meta

```
the `<build><disable/></build>` part is missing.

Further Information
===============
* A link to an OBS instance showing the issue
* Exact server version (in case a private instance is used)

**Comment by adrianschroeter** [comment]
it is also missing in the backend storage btw.

**Comment by eduardoj** [comment]
> As @lethliel found out we loose flags on project creation. I am relative sure that this is a regression and it should be fixed before 2.10.

Confirmed for 2.10. Confirmed for 2.9 too, so it is not a regression.

**Comment by eduardoj** [comment]
> > As @lethliel found out we loose flags on project creation. I am relative sure that this is a regression and it should be fixed before 2.10.
> 
> Confirmed for 2.10. Confirmed for 2.9 too, so it is not a regression.

I checked today again. Issue confirmed for 2.10. It does not happen in 2.9. So it is a regression. Yesterday I tested too fast.

**Comment by coolo** [comment]
@Ana06 and I bisected the problem after running in circles while debugging.

```
commit 4097a4754c1bf96d681fc6fd535acefdc208043c
Author: depfu[bot] <depfu[bot]@users.noreply.github.com>
Date:   Thu Aug 23 09:42:25 2018 +0000

    Update rails to version 5.2.1
